### DISCUSSION TITLE
Resizing existing paths in PDF

### DISCUSSION THREAD

**Discussion by qwertynik** [pymupdf/PyMuPDF#1722] [Looking for help]
What are the ways using which existing paths in the same PDF (or new PDF) can be resized?
GIF:
![ShapeResizing](https://user-images.githubusercontent.com/6836536/169476442-76361707-1993-4dd2-846c-71a3ff4693e6.gif)

**Comment by JorjMcKie** [comment]
You always can redraw paths on another page.
When doing this, you can apply a transformation to the coordinates, i.e. multiply them by some matrix.
Or even simpler, take the original page (or a clip within it) and use `newpage.show_pdf_page()` with the desired target rectangle and rotation.

**Reply by qwertynik** [reply]
Had read your response earlier. Will implement the suggestion and post the outcome here at the earliest possible.

**Reply by 94LT** [reply]
> You always can redraw paths on another page. When doing this, you can apply a transformation to the coordinates, i.e. multiply them by some matrix. Or even simpler, take the original page (or a clip within it) and use `newpage.show_pdf_page()` with the desired target rectangle and rotation.

Whether you can change its size or remove it on the current page？look forward to your reply

**Reply by JorjMcKie** [reply]
> Whether you can change its size or remove it on the current page？

No, this is not supported by PyMuPDF.
